General/general_shacl.ttl

```diff
@@ -0,0 +1,206 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:generalShape a sh:NodeShape ;
+    sh:property [ sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:minCount "1"^^xsd:unsignedInt ;
+            sh:node general:DescriptionShape ;
+            sh:order "0"^^xsd:unsignedInt ;
```

Comment:
as of specification order and minCount is an integer and not unsignedInt.
Changing this to integer you eliminate many warnings in a IDE.

But I guess you can remove that part including and after ^^^ at all.

Reply:
fixed